docs/integrations/destinations/bigquery.md

```diff
@@ -140,6 +140,7 @@ Now that you have set up the BigQuery destination connector, check out the follo
 
 | Version | Date       | Pull Request                                               | Subject                                                                                                                                                         |
 |:--------|:-----------|:-----------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 2.3.6   | 2023-11-03 | [32125](https://github.com/airbytehq/airbyte/pull/32125)   | fix compilation warnings                                                                                                                                         |
```

Comment:
nit: here and in the other changelogs, I think the significant change is rather that we're adopting version 0.4.0 of the CDK.

I'm assuming that these changelogs are read when the connector's misbehaving after an upgrade; knowing about the CDK change is probably going to be more relevant in that use case.